app/uk/gov/hmrc/api/testlogin/connectors/AuthLoginStubConnector.scala

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2016 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.api.testlogin.connectors
+
+import javax.inject.Inject
+
+import play.api.libs.ws.WSClient
+import play.api.mvc.Session.COOKIE_NAME
+import uk.gov.hmrc.api.testlogin.models.{TestOrganisation, TestIndividual, TestUser}
+import uk.gov.hmrc.crypto.ApplicationCrypto.SessionCookieCrypto
+import uk.gov.hmrc.crypto.Crypted
+import uk.gov.hmrc.domain.{EmpRef, SaUtr}
+import uk.gov.hmrc.play.config.ServicesConfig
+import uk.gov.hmrc.play.frontend.auth.connectors.domain.ConfidenceLevel
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+trait AuthLoginStubConnector {
+  val serviceUrl: String
+  val wsClient: WSClient
+
+  def createSession(testUser: TestUser): Future[String] = {
+    val form = GovernmentGatewayForm.from(testUser)
+
+    wsClient.url(s"$serviceUrl/auth-login-stub/gg-sign-in").withFollowRedirects(false).post(form.toMap) map { response =>
+      response.status match {
+        case 303 =>
```

Comment:
Instead of a magic number, here we could use `play.api.http.Status.SEE_OTHER`. Similarly for all other status codes in this pull request.